src/main/scala/ai/eto/rikai/sql/model/fs/FileSystemRegistry.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Rikai authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.eto.rikai.sql.model.fs
+
+import ai.eto.rikai.sql.model.{Model, ModelNotFoundException, Registry}
+import ai.eto.rikai.sql.spark.Python
+import org.apache.logging.log4j.scala.Logging
+
+/**
+  * FileSystem-based Model [[Registry]].
+  */
+class FileSystemRegistry(val conf: Map[String, String])
+    extends Registry
+    with Logging {
+
+  private val pyClass: String = "rikai.spark.sql.codegen.fs.Registry"
```

Comment:
All of these raw string references are going to suck if we have to refactor/move packages. Not sure what's a better alternative tho.

Reply:
Yea, i agree. But this is also give the flexibility that if a third party wanted to add anything (i.e., extensions) to a released `rikai`, it is relatively easy.